### ISSUE TITLE
There are issues with reproducing the results

### ISSUE THREAD

**Issue by zhang-bofan** [Liu-Yating/CSKT#1]
I just run the train.py:

Total Param: 163.00M Trainable Param: 14.00 M
Using 5.0 times learning rate for random init module 
2025-03-31 17:41:59,606 IRRA.eval INFO: 
+------+-------+-------+-------+-------+-------+
| task |   R1  |   R5  |  R10  |  mAP  |  mINP |
+------+-------+-------+-------+-------+-------+
| t2i  | 0.300 | 2.400 | 4.950 | 1.206 | 0.679 |
+------+-------+-------+-------+-------+-------+
2025-03-31 17:41:59,606 IRRA INFO: test done. Time: 5.264[s]
2025-03-31 17:41:59,607 IRRA.train INFO: start training
2025-03-31 17:42:48,185 IRRA.train INFO: Epoch[1] Iteration[100/579], loss: 36.5218, sdm_loss: 28.3054, id_loss: 8.2164, img_acc: 0.0002, txt_acc: 0.0002, Base Lr: 1.00e-06
2025-03-31 17:43:34,697 IRRA.train INFO: Epoch[1] Iteration[200/579], loss: 36.4978, sdm_loss: 28.2813, id_loss: 8.2165, img_acc: 0.0002, txt_acc: 0.0003, Base Lr: 1.00e-06
2025-03-31 17:44:20,727 IRRA.train INFO: Epoch[1] Iteration[300/579], loss: 36.4760, sdm_loss: 28.2595, id_loss: 8.2166, img_acc: 0.0003, txt_acc: 0.0003, Base Lr: 1.00e-06
2025-03-31 17:45:06,656 IRRA.train INFO: Epoch[1] Iteration[400/579], loss: 36.4575, sdm_loss: 28.2408, id_loss: 8.2166, img_acc: 0.0003, txt_acc: 0.0003, Base Lr: 1.00e-06
2025-03-31 17:45:52,939 IRRA.train INFO: Epoch[1] Iteration[500/579], loss: 36.4421, sdm_loss: 28.2254, id_loss: 8.2167, img_acc: 0.0003, txt_acc: 0.0002, Base Lr: 1.00e-06
2025-03-31 17:46:29,486 IRRA.train INFO: Epoch 1 done. Time per batch: 0.455[s] Speed: 140.5[samples/s]
2025-03-31 17:46:29,486 IRRA.train INFO: Epoch 1 done. Time total : 263.699[s]
2025-03-31 17:46:29,486 IRRA.train INFO: Validation Results - Epoch: 1
2025-03-31 17:46:34,751 IRRA.eval INFO: 
+------+-------+-------+-------+-------+-------+
| task |   R1  |   R5  |  R10  |  mAP  |  mINP |
+------+-------+-------+-------+-------+-------+
| t2i  | 0.550 | 2.300 | 4.850 | 1.202 | 0.664 |
+------+-------+-------+-------+-------+-------+

Is there any problem?

**Comment by Liu-Yating** [comment]
No problem. Clear improvements can be observed after a few epochs.
